File: src/main/java/com/trading/kalyani/KTManager/model/Message.java
package com.trading.kalyani.KTManager.model;

import com.google.gson.annotations.SerializedName;
import lombok.*;

import java.util.Date;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@ToString

public class Message {

    @SerializedName("instrumentToken")
    private long instrumentToken;

    @SerializedName("oi")
    private double oi;

    @SerializedName("lastTradedTime")
    private Date lastTradedTime;

    @SerializedName("lastTradedPrice")
    private double lastTradedPrice;

    @SerializedName("tickTimestamp")
    private Date tickTimestamp;

    @Override
    public int hashCode() {
        return super.hashCode();
    }
}

File: kpn-ui/src/app/models/message.ts

export class Message {
    instrumentToken: number;
    oi: number;
    lastTradedPrice:number
    lastTradedTime: Date;
    tickTimestamp: Date;

    constructor(instrumentToken: number, oi: number, lastTradedPrice: number,lastTradedTime: Date,tickTimestamp: Date) {
        this.instrumentToken = instrumentToken;
        this.oi = oi;
        this.lastTradedPrice = lastTradedPrice;
        this.lastTradedTime = lastTradedTime;
        this.tickTimestamp = tickTimestamp;
      }
}